src/main/scala/viper/gobra/frontend/Parser.scala

```diff
@@ -1207,19 +1208,14 @@ object Parser {
 
     lazy val access : Parser[PAccess] =
       "acc" ~> "(" ~> expression <~ ")" ^^ { exp => PAccess(exp, PFullPerm().at(exp)) } |
-      "acc" ~> "(" ~> expression ~ ("," ~> permission <~ ")") ^^ PAccess
+      // parsing wildcard permissions should be done here instead of in [[permission]] to avoid parsing "_"
+      // as an expression in arbitrary parts of the code
+      "acc" ~> "(" ~> expression <~ ("," ~> wildcard <~ ")") ^^ { exp => PAccess(exp, PWildcardPerm().at(exp)) } |
+      "acc" ~> "(" ~> expression ~ ("," ~> expression <~ ")") ^^ PAccess
 
     lazy val permission: Parser[PPermission] =
-      fractionalPermission |
-      "write" ^^^ PFullPerm() |
-      "none" ^^^ PNoPerm() |
-      "_" ^^^ PWildcardPerm()
-
-    lazy val fractionalPermission: Parser[PFractionalPerm] =
-      expression into {
-        case d@PDiv(left, right) => success(PFractionalPerm(left, right).at(d))
-        case e => failure(s"expected a fractional permission amount expressed as a division but got $e")
-      }
+      "writePerm" ^^^ PFullPerm() |
```

Comment:
Did we decide to change these keywords? If so, I think you'll need to update the list of keywards accordingly in this file

Reply:
I did not discuss this with any one. However, because we can have `perm` expressions in any place where a "regular" expression may occur, this gives rise to ambiguities when we use `none` for representing permission `0`, e.g.
```go
whatAmI := none
```
In this case, the type of `whatAmI` is ambiguous: it can either be `option[???]` or `perm`. Instead of complicating the type system to infer a default type when it cannot be obtained from the context, I opted to change the keyword from `none` to `noPerm`. The change from `write` to `writePerm` was done for consistency.